src/Aquifer.Jobs/Subscribers/TranslationMessageSubscriber.cs

```diff
@@ -71,13 +78,24 @@ await TranslateResourceCoreAsync(
     ///     which requires using durable functions to maintain state.
     ///     (see https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-bindings#durableTaskClient-usage).
     /// </summary>
-    [Function(nameof(TranslateProjectResources))]
-    public async Task TranslateProjectResources([QueueTrigger(Queues.TranslateProjectResources)] QueueMessage queueMessage,
+    [Function(nameof(TranslateProjectResourcesMessageSubscriber))]
+    public async Task TranslateProjectResourcesMessageSubscriber([QueueTrigger(Queues.TranslateProjectResources)] QueueMessage queueMessage,
         [DurableClient] DurableTaskClient durableTaskClient,
-        CancellationToken ct)
+        CancellationToken cancellationToken)
```

Comment:
What's the reason for changing this here? We generally call it `ct` everywhere else.

Reply:
Oh I guess you want it on line 89